doc/source/structures/collections/iterator.rst

```diff
@@ -40,15 +40,8 @@ invalid.  This represents a position just *before the start* of the list of
 items.  Only after the first time :attr:`NEXT` is called does the value of
 :attr:`VALUE` become usable as the first thing in the collection.
 
-Rewinding No Longer Supported
------------------------------
-
-.. note::
-
-    There used to be a :RESET method for iterators, but it has been
-    removed as it was not always implemented and sometimes gave an
-    error.  Now to start the enumeration over you need to obtain a
-    new iterator.
+If you want to restart an iteration you must call the ``:ITERATOR`` suffix of
+the collection again to obtain a new iterator.
```

Comment:
Disagree, because the suffix being missing can break old scripts.  Maybe go ahead and remove the subject heading ("Rewinding No Longer Supported"), but don't delete the fact that :RESET used to exist and doesn't anymore.  Maybe lead with the new text (the +" text lines in the git diff here), but then add a sentence that says there used to be a :RESET suffix but it was removed because it couldn't be supported on all types.

Reply:
Works for me!